bindings-python/setup.py

```diff
@@ -22,13 +23,25 @@ def requirements():
             yield line
 
 
-setuptools.setup(
-    name='gardener-component-model',
-    version=version(),
-    description='Gardener Component Model',
-    python_requires='>=3.8.*',
-    packages=setuptools.find_packages(),
-    install_requires=list(requirements()),
-    entry_points={
-    },
-)
+@contextlib.contextmanager
+def prepare_environment():
+    json_spec_target_path = os.path.join(own_dir, 'gci', 'jsonschema')
+    os.symlink(
+        os.path.join(own_dir, '..', 'language-independent'),
+        json_spec_target_path,
+    )
+    yield
+    os.unlink(json_spec_target_path)
+
+
+with prepare_environment():
+    setuptools.setup(
+        name='gardener-component-model',
+        version=version(),
+        description='Gardener Component Model',
+        python_requires='>=3.8.*',
+        packages=setuptools.find_packages(),
+        package_data={'gci': ['jsonschema/component-descriptor-v2-schema.yaml']},
+        install_requires=list(requirements()),
+        entry_points={},
```

Comment:
how about adding: `validate-component-descriptor = gci/validate:main` to entry_points (see cc-utils/cli/setup.py)?